actors/rozetka-daily/src/consts.js

```diff
@@ -0,0 +1,42 @@
+export const LABELS = {
```

Comment:
This second actor looks like most of the code is copy pasted from the first one. That will make it harder for maintenance. I would use a single folder for those and just changed the behaviour via input or env var. cc @rarous

Reply:
Yeah, it look like having separate actors for count functionality is not good idea.

Reply:
Well it could be multiple actors but definitely should not be multiple folders. But I will leave that up to you.

Reply:
I guess, I would be the most efficient to do the count of the results and scraping the data in parallel, considering the crawling logic there has to be very similar. I just don't understand clearly how we should handle the output in case of having one actor for both purposes (though there are plenty ways how handle this, for example we could save the results to separate named datasets for count the the daily actors). But the reuse of the code is great, and I'm very glad that I'm allowed to do that here.
So, to sum up, which way should I choose — separate actors with some shared code or a single actor (some more specs about the output should be provided here then)?

Reply:
We have already `common` library for reusable code, but it should be for code that is reusable for all/most of actors. 

This case should be IMHO handled just by `type: "COUNT"` Input parameter. It will be one actor in more modes (we already have this for Black Friday scraping in older actors). It will be scheduled with different input parameters. This mode will just write to Dataset but skip the Keboola upload step.

Sorry, for inconvenience, we are still figuring out the process and shape - count functionality is new requirement for internal benchmarking of scraped data.

Reply:
I agree with @rarous 's approach, will be the simplest

Reply:
Hey, I guess it is not properly explained in the docs, but I dont think it makes sense to have an actor four count which does the same as the main actor. The idea behind it is to double check the result, i.e. find and resonably use the numbers which tels how many items there are in each category, like with rozetka here: 
![image](https://user-images.githubusercontent.com/39551460/115395028-b61e0180-a1e3-11eb-9c06-cc66f51a980f.png)